redbot/cogs/audio/core/events/lavalink.py

```diff
@@ -227,6 +228,20 @@ async def lavalink_event_handler(
                 player_check = await self.get_active_player_count()
                 await self.update_bot_presence(*player_check)
 
+        if event_type == lavalink.LavalinkEvents.TRACK_START and vc_status:
+            log.debug("Track started for %s, updating voice channel status", guild_id)
+            voice_channel = current_channel
+            track_title = await self.get_current_track_title()
+            await self.update_voice_channel_presence(voice_channel, track_title)
+
+        if event_type == lavalink.LavalinkEvents.TRACK_END and vc_status:
+            await asyncio.sleep(1)
+            if not player.is_playing:
+                log.debug("Track ended for %s, updating voice channel status", guild_id)
+                voice_channel = current_channel
+                track_title = await self.get_current_track_title()
+                await self.update_voice_channel_presence(voice_channel, track_title)
+
```

Comment:
I think merging the code from these two `if` statements with the code right above might be a good idea. The coed doesn't seem *that* different + we'd avoid an additional `sleep` call.